.github/workflows/asv-pr.yml

```diff
@@ -6,8 +6,8 @@
 name: Run benchmarks for PR
 
 on:
-  pull_request:
-    branches: [ main ]
+  # pull_request:
```

Comment:
Is this dead code?  If there's a case when it needs to return, maybe a comment above the block to say.

Reply:
Oopsy kitties! I commented out some other tests so CI would run faster for iterating on the docker solution. Un-commented.